ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -251,7 +253,7 @@ public static Bytes32 get_seed(BeaconState state, UnsignedLong epoch, Bytes4 dom
     UnsignedLong randaoIndex =
         epoch.plus(UnsignedLong.valueOf(EPOCHS_PER_HISTORICAL_VECTOR - MIN_SEED_LOOKAHEAD - 1));
     Bytes32 mix = get_randao_mix(state, randaoIndex);
-    final Bytes epochBytes = int_to_bytes(epoch.longValue(), 8);
+    Bytes epochBytes = int_to_bytes(epoch.longValue(), 8);
     return Hash.sha2_256(Bytes.concatenate(domain_type.getWrappedBytes(), epochBytes, mix));
```

Comment:
I think the order of element concatenation is wrong here. https://github.com/ethereum/eth2.0-specs/blob/v0.9.0/specs/core/0_beacon-chain.md#get_seed